build/prepare-versioning.ps1

```diff
@@ -35,9 +35,9 @@ $nugetVersion = "{0}.{1}.{2}" -f ($versionTokens[0], $versionTokens[1], $version
 
 Write-Host "=========================="
 Write-Host "Versioning .nuspec file..."
-$filename = $env:TO_PACK_TARGET + "\" + $env:LIBRARY_NAME + ".nuspec"
+$filename = "build\Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec"
```

Comment:
Update the source _nuspec_ file in place (the script updates the common properties file in-place the same way).
The reason for this change (apart from making the script internally consistent) is that it makes it self-contained - it no longer depends on having external variables set.